app/models/user.rb

```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] It would be great if you remove line 3 which is a commented piece of code 😨 Kindly remove it to make your code clean and look professional 😊